python/mxnet/symbol/symbol.py

```diff
@@ -1437,6 +1437,13 @@ def _gen_atomic_symbol(self):
         return Symbol(handle)
 
 
+    def partition(self, name):
```

Comment:
- rename to "optimizeFor"
- return a copy of the modified input symbol

Reply:
Done!